markdown/source_md/types-and-typeclasses.md

```diff
@@ -424,7 +424,6 @@ It has a type declaration of `fromIntegral :: (Num b, Integral a) => a -> b`.
 From its type signature we see that it takes an integral number and turns it into a more general number.
 That's useful when you want integral and floating point types to work together nicely.
 For instance, the `length` function has a type declaration of `length :: [a] -> Int` instead of having a more general type of `(Num b) => length :: [a] -> b`.
-I think that's there for historical reasons or something, although in my opinion, it's pretty stupid.
 Anyway, if we try to get a length of a list and then add it to `3.2`, we'll get an error because we tried to add together an `Int` and a floating point number.
```

Comment:
To maintain continuity:
```suggestion
If we try to get a length of a list and then add it to `3.2`, we'll get an error because we tried to add together an `Int` and a floating point number.
```